lib/src/main/java/com/rey/material/util/ColorUtil.java

```diff
@@ -8,13 +8,13 @@ private static int getMiddleValue(int prev, int next, float factor){
 		return Math.round(prev + (next - prev) * factor);
 	}
 	
-	public static int getMiddleColor(int prevColor, int curColor, float factor){		
+	public static int getMiddleColor(int prevColor, int curColor, float factor, boolean disableOnMinimumValue){
```

Comment:
I don't think we should add a new parameter to this method because It isn't common case. We should handle the condition in Slider class.